src/garage/torch/algos/ppo.py

```diff
@@ -48,7 +51,7 @@ def __init__(self,
                  policy,
                  baseline,
                  optimizer=torch.optim.Adam,
-                 policy_lr=_Default(3e-4),
+                 policy_lr=3e-4,
                  max_path_length=500,
```

Comment:
this is extremely large. 100-150 is more typical

Reply:
fixed